src/components/shared/MarkerContextMenu.js

```diff
@@ -258,61 +260,63 @@ class MarkerContextMenuImpl extends PureComponent<Props> {
+          disabled={this._isZeroDurationMarker(marker)}
+        >
+          Set selection from marker’s duration
+        </MenuItem>
+        <div className="react-contextmenu-separator" />
         {this._isZeroDurationMarker(marker) ? (
           <>
             <MenuItem onClick={this.setStartRangeFromMarkerStart}>
-              Set selection start time here
+              Start selection here
             </MenuItem>
             <MenuItem onClick={this.setEndRangeFromMarkerEnd}>
-              Set selection end time here
+              End selection here
             </MenuItem>
           </>
         ) : (
           <>
-            <SubMenu title="Set selection start">
-              <MenuItem onClick={this.setStartRangeFromMarkerStart}>
-                From the start of this marker
-              </MenuItem>
-              <MenuItem
-                onClick={this.setStartRangeFromMarkerEnd}
-                disabled={markerEnd > selectionEnd}
-              >
-                From the end of this marker
-              </MenuItem>
-            </SubMenu>
-            <SubMenu title="Set selection end">
-              <MenuItem
-                onClick={this.setEndRangeFromMarkerStart}
-                disabled={selectionStart > markerStart}
-              >
-                From the start of this marker
-              </MenuItem>
-              <MenuItem onClick={this.setEndRangeFromMarkerEnd}>
-                From the end of this marker
-              </MenuItem>
-            </SubMenu>
+            <MenuItem onClick={this.setStartRangeFromMarkerStart}>
+              Start selection at marker’s <strong>start</strong>
+            </MenuItem>
+            <MenuItem
+              onClick={this.setStartRangeFromMarkerEnd}
+              disabled={markerEnd > selectionEnd}
+            >
+              Start selection at marker’s <strong>end</strong>
+            </MenuItem>
+            <div className="react-contextmenu-separator" />
+            <MenuItem
+              onClick={this.setEndRangeFromMarkerStart}
+              disabled={selectionStart > markerStart}
+            >
+              End selection at marker’s <strong>start</strong>
+            </MenuItem>
+            <MenuItem onClick={this.setEndRangeFromMarkerEnd}>
+              End selection at marker’s <strong>end</strong>
+            </MenuItem>
           </>
         )}
 
-        <MenuItem
-          onClick={this.setRangeByDuration}
-          disabled={this._isZeroDurationMarker(marker)}
-        >
-          Set selection from duration
-        </MenuItem>
+        <div className="react-contextmenu-separator" />
         <MenuItem onClick={this.copyMarkerDescription}>
           Copy description
         </MenuItem>
         {data && data.cause ? (
-          <MenuItem onClick={this.copyMarkerCause}>Copy marker cause</MenuItem>
+          <MenuItem onClick={this.copyMarkerCause}>
+            Copy attached stack
```

Comment:
Hm, I'm not sure about the "attached" here. But not sure what would be a better wording either. Maybe "Copy marker's stack" or just "Copy stack". I guess latter would be better since we already have "Copy description". But as I said, I'm also okay with the current wording if you feel like they are not better :) What do you think?

Reply:
I think I discussed this with @fqueze while I was working on this. But we should maybe ask an native english speaker :) I'll try to find someone.

Reply:
Changed to `Copy call stack`